test/unit/new-utils.js.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+const { assert } = require("chai");
+const { bigintToLong, longToBigint } = require("../../lib/new-utils");
+const Long = require("long");
+
```

Comment:
I assume double `.js` in `new-utils.js.js` is on purpose, as it's a `.js` file that tests `new-utils.js`.